client/components/errorLabel/errorLabel.css

```diff
@@ -1,4 +1,4 @@
-.button-error {
+.error {
     display: flex;
     flex-direction: row;
     align-items: center;
```

Comment:
Very generic class name, can clash with other classes with same name. Better to name is same as component `.errorLabel`